src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Display from './Display';
+import Key from './Key';
+import './Calculator.css';
+
+function Calculator() {
```

Comment:
- [OPTIONAL] What is your opinion about using the new arrow function syntax 💭? It is more concise and easier to read

Reply:
Ok, I try to use it, thanks @wayungi.